packages/rum/test/viewCollection.spec.ts

```diff
@@ -193,6 +211,21 @@ describe('rum track url change', () => {
     window.location.hash = '#bar'
   })
 
+  it('should not create a new view when it is an Anchor navigation', (done) => {
```

Comment:
It could be nice to have e2e tests to validate the behavior with and without anchor